docs/NF_Dev.md

```diff
@@ -99,17 +103,58 @@ values.
 }
 ```
 
+Additionally, a developer can run the [config python script](../examples/config.py) to deploy multiple network functions, including linear or circular chains of multiple NFs, from a JSON config file. An example config file can be found [here](../examples/example_nf_deploy.json). In the config file, a user can specify a number of seconds for the NFs to run before shutdown with the global "TTL". If no timeout is specified, the NFs will run until a raised error or a manual shutdown (Ctrl + C).
+When the NFs are launched, if the user specifies a directory name with the global "directory", a directory will be created (if it does not already exist) with the user-specified name. If a global "directory-prefix" is specified, a directory will be created with the specified prefix + timestamp. If no directory name or prefix is specified, the default name of the created directory will the be the timestamp. Output from each NF will be continuously written to the corresponding log text file within the created or pre-existing directory. Format of the log file name is as follows: "log-NF name-instance ID". To track the output of a NF:
+
+```
+tail -f <logfile name>
+```
+
+This script must be run within the /examples folder:
+
+```
+python3 config.py <config file name>
+```
+
+### JSON Config File For Deploying Multiple NFs
+
+```
+{
+   	"globals [optional]": [
+		{
+			"TTL": X (seconds)
+		},
+	        {
+      			"directory": "existing or new dir name"
+   		}
+	],
+	"NF name": [
```

Comment:
have you tested that the program fails cleanly with a useful error if you give it a junk "NF Name" parameter?

Reply:
Current behavior is that it will exit and fail with the print statement: "Error: Unable to access NF directory <NF NAME>. Check syntax in your configuration file"

Reply:
Ok that's fine.